.github/workflows/main.yml

```diff
@@ -14,5 +14,7 @@ jobs:
       - name: Install Node.js dependencies
         run: npm ci
 
+      - name: File metadata
+        run: npm run check:metadata
```

Comment:
CI has dedicated steps for each check, rather than simply running `npm run check`, because it makes the output more structured.